src/Filter.js

```diff
@@ -92,13 +95,16 @@ function Filter({
 
   const resetFilter = function () {
     // reset the inputstates to one if multiple exist
-    if (inputStates.length > 1) {
-      setInputStates([{ columnName: [''], logicValue: [], inputValue: [] }])
+    if (inputStates[0].inputValue.length) {
```

Comment:
@cotts could you explain why this is needed?

Reply:
because the filter array will always contain 1 item and also the `inputValue` will not be erased when resetting filter if there is only one filter... this fix resets the filter when there are one or more filters applied.

Reply:
@cotts is it related to disabling copy button?

Reply:
it's a minor bug fix that I found while I was trying to manage the disable button.